Makefile

```diff
@@ -29,6 +29,8 @@ export DOCKER_BUILD_FLAGS =
 ifeq ($(wildcard /.dockerenv),)
 export DOCKER_RUN_FLAGS ?= -it --rm
 export DOCKER_RUN := docker run $(DOCKER_RUN_FLAGS) -v $(CURDIR):/src -p $(HUGO_PORT):$(HUGO_PORT) -e YARN_BUILD_DISABLED -e GITHUB_BASIC_AUTH $(DOCKER_IMAGE_NAME)
+export DOCKER_ALGOLIA_ENVS ?= -e ALGOLIA_INDEX_NAME -e ALGOLIA_APP_ID -e ALGOLIA_ADMIN_KEY -e ALGOLIA_INDEX_FILE
```

Comment:
If we had to do this for algolia, we probably need it for cloudflare

Reply:
I assumed so, too, but the cloudflare target apparently ran with only a few warnings and no errors.

Reply:
https://github.com/cloudposse/docs/runs/1843433923

Reply:
https://github.com/danielpigott/cloudflare-cli/issues/38

Reply:
The warning is indicative of not being able to read the envs. Unfortunately, in the world of software engineering, error checking and documentation uncommon practices. ;)

Reply:
(we can also upgrade the cloudflare-cli while we're at it)